Description: Move @itemize after @subsection to satisfy texinfo-5.1
Author: Bryan Hundven <bryanhundven@gmail.com>
Origin: upstream, http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4813
Forwarded: not-needed
Applied-Upstream: http://bazaar.launchpad.net/~vcs-imports/grub/grub2-bzr/revision/4813
Last-Update: 2013-08-11

Index: b/docs/grub-dev.texi
===================================================================
--- a/docs/grub-dev.texi
+++ b/docs/grub-dev.texi
@@ -1394,8 +1394,8 @@
 
 @node Bitmap API
 @section Bitmap API
-@itemize
 @subsection grub_video_bitmap_create
+@itemize
 @item Prototype:
 @example
 grub_err_t grub_video_bitmap_create (struct grub_video_bitmap **bitmap, unsigned int width, unsigned int height, enum grub_video_blit_format blit_format)
